contracts/utils/EnumerableSet.sol

```diff
@@ -84,44 +88,121 @@ library EnumerableSet {
     /**
      * @dev Returns true if the value is in the set. O(1).
      */
-    function contains(AddressSet storage set, address value)
-        internal
-        view
-        returns (bool)
-    {
+    function contains(Set storage set, bytes32 value) private view returns (bool) {
         return set._indexes[value] != 0;
     }
 
     /**
-     * @dev Returns an array with all values in the set. O(N).
+     * @dev Returns the number of values on the set. O(1).
+     */
+    function length(Set storage set) private view returns (uint256) {
```

Comment:
Would you mind commenting on this? `length` doesn't sound right to me.

Reply:
I think this is highly subjective and ultimately decided by context: in C++ for instance `vector` has `size`, as does every other STL data structure, but in Rust `vec` has `len`, as do `HashMap` and `HashSet`. 

I chose `length` because its what Solidity uses for its array types (both `[]` and `bytes`).